homeassistant/components/notify/xmpp.py

```diff
@@ -34,76 +34,92 @@
 })
 
 
-def get_service(hass, config, discovery_info=None):
+async def async_get_service(hass, config, discovery_info=None):
     """Get the Jabber (XMPP) notification service."""
+    _LOGGER.debug("async_get_service")
```

Comment:
Looks like a debug messages for the testing during the development.

Reply:
yes, will remove